project/plugins.sbt

```diff
@@ -10,7 +10,7 @@ addSbtPlugin("com.github.sbt"                    % "sbt-native-packager"      %
 addSbtPlugin("com.github.sbt"                    % "sbt-github-actions"       % "0.15.0")
 addSbtPlugin("com.github.sbt"                    % "sbt-pgp"                  % "2.2.1")
 addSbtPlugin("com.github.sbt"                    % "sbt-release"              % "1.1.0")
-addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"             % "0.10.4")
+addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"             % "0.11.0")
```

Comment:
Need to update this as well because new Scala version is only compatible with newer plugin